shap/plots/_image.py

```diff
@@ -9,10 +9,12 @@
     import matplotlib.pyplot as pl
 except ImportError:
     warnings.warn("matplotlib could not be loaded!")
-    pass
+try:
+    from IPython.core.display import display, HTML
+except ImportError:
+    warnings.warn("IPython could not be loaded!")
```

Comment:
Instead of using try-except, we could lazily import `dipslay` and `HTML` in `image` and `image_to_text`:

```python
def image(...):
    import from IPython.core.display import display, HTML


def image_to_text(...):
    from IPython.core.display import display, HTML
```

Reply:
I think it is clearer to leave it at the top for now. If the computation becomes a problem then we can move it later.